test-infra/src/main/java/com/datastax/oss/driver/api/testinfra/ccm/CcmBridge.java

```diff
@@ -324,7 +324,11 @@ public void create() {
       }
       if (getCassandraVersion().compareTo(Version.V2_2_0) >= 0 && !SCYLLA_ENABLEMENT) {
         // @IntegrationTestDisabledScyllaJVMArgs @IntegrationTestDisabledScyllaUDF
-        updateConfArguments.append("enable_user_defined_functions:true").append(' ');
+        if (getCassandraVersion().compareTo(Objects.requireNonNull(Version.parse("4.1.0"))) >= 0) {
```

Comment:
Minor nit: `Version.parse("4.1.0")` could be extracted to a constant (like `Version.V2_2_0`). Additionally `Objects.requireNonNull` should not be needed.

Reply:
Done. 
Will push a new ver in a minute.